package.json

```diff
@@ -10,10 +10,11 @@
     "open:src": "babel-node tools/srcServer.js",
     "lint": "esw webpack.config.* src tools",
     "lint:watch": "npm run lint -- --watch",
-    "clean-dist": "npm run remove-dist && mkdir ./lib/jekyll-admin/public",
+    "clean-dist": "npm run remove-dist  && npm run add-dist",
+    "add-dist": "mkdirp ./lib/jekyll-admin/public",
     "remove-dist": "rimraf ./lib/jekyll-admin/public",
     "build:html": "babel-node tools/buildHtml.js",
-    "prebuild": "npm run clean-dist && npm run build:html && npm run lint && npm test",
```

Comment:
I think we do need to lint and test before building just to double check everything as expected.